.github/workflows/sdk-release.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
       - run: cd ./ecosystem/typescript/sdk && pnpm build
 
       # Ensure the version, changelog, etc. are valid.
-      - run: cd ./ecosystem/typescript/sdk && ./check.sh
+      - run: cd ./ecosystem/typescript/sdk && ./scripts/check.sh
```

Comment:
instead of doing `cd` in each step, you can use `workding-directory`

Reply:
I can work on it in a different PR, we might change the whole process and stop using this script. 
but would like to get this change in just to not break current devnet branch CI

Reply:
sure. stamping to unblock then!